<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveStudent</class>
 <widget class="QWidget" name="RemoveStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>561</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Remove Studnet form Class</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>90</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Student</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Class</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Studnet name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>561</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Remove Full Class With All Students</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Class</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selet Class</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Studnt</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Class</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>RemoveStudent</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>817</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>RemoveStudent</receiver>
   <slot>removeStudentRadio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>-80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>RemoveStudent</receiver>
   <slot>removeClassRedio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>-80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RemoveStudent</receiver>
   <slot>removeStudent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>RemoveStudent</receiver>
   <slot>removeClass()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>813</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeStudentRadio()</slot>
  <slot>removeClassRedio()</slot>
  <slot>removeStudent()</slot>
  <slot>removeClass()</slot>
 </slots>
</ui>
